src/server/scripts/Kalimdor/zone_tanaris.cpp

```diff
@@ -84,7 +84,7 @@ class npc_aquementas : public CreatureScript
                 ItemPosCountVec dest;
                 uint8 msg = player->CanStoreNewItem(NULL_BAG, NULL_SLOT, dest, 11522, 1, NULL);
                 if (msg == EQUIP_ERR_OK)
-                    player->StoreNewItem(dest, 11522, 1, true);
```

Comment:
Was it an error ? Because `player->StoreNewItem(ItemPosCountVec, uint32, int32, bool)` doesn't exist

Reply:
Actually it exists: Item* StoreNewItem(ItemPosCountVec const& pos, uint32 item, bool update, int32 randomPropertyId = 0);

Reply:
Item* StoreNewItem(ItemPosCountVec const& pos, uint32 item, **bool update, int32 randomPropertyId = 0**); exists yes
but not 
Item* StoreNewItem(ItemPosCountVec const& pos, uint32 item, **int32 randomPropertyId = 0, bool update**);
Look at the order of parameters

Reply:
Oh, yes, my bad. Maybe it didn't threw an error because the true was considered an 1 and the function was:
```cpp
player->StoreNewItem(dest, 11522, 1, 1)
```
strange..